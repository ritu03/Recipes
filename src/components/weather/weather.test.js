import React from 'react'
import { render, cleanup  } from '@testing-library/react'
import configureMockStore from 'redux-mock-store'
import WeatherWidget from './weather'

const mockStore = configureMockStore()
const storeValues = {
  weatherData: {
    data: null,
    status: ''
  }
}
const store = mockStore(storeValues)
describe('WeatherWidget snapshot', () => {
  afterEach(cleanup)

  test('renders properly', () => {
    const { container } = render(<WeatherWidget store={store} />)
    expect(container).toMatchSnapshot()
  })
  })