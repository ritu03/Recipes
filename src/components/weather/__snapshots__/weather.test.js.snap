// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeatherWidget snapshot renders properly 1`] = `
<div>
  <div
    class="sc-eCApnc dwLuXt"
  >
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-grid-xs-12"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-center MuiGrid-justify-xs-center MuiGrid-grid-xs-12 MuiGrid-grid-md-4"
      >
        <div
          class="sc-iCoGMd hgXSwb"
        >
          <div
            class="MuiFormControl-root MuiTextField-root"
          >
            <label
              class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined MuiFormLabel-filled"
              data-shrink="true"
              for="outlined-basic"
              id="outlined-basic-label"
            >
              Enter City
            </label>
            <div
              class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
            >
              <input
                aria-invalid="false"
                class="MuiInputBase-input MuiOutlinedInput-input"
                id="outlined-basic"
                type="text"
                value="Melbourne"
              />
              <fieldset
                aria-hidden="true"
                class="PrivateNotchedOutline-root-1 MuiOutlinedInput-notchedOutline"
              >
                <legend
                  class="PrivateNotchedOutline-legendLabelled-3 PrivateNotchedOutline-legendNotched-4"
                >
                  <span>
                    Enter City
                  </span>
                </legend>
              </fieldset>
            </div>
          </div>
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiButton-label"
            >
              Check Weather
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-center MuiGrid-justify-xs-center MuiGrid-grid-xs-12 MuiGrid-grid-md-6"
      >
        <fieldset
          class="MuiFormControl-root"
        >
          <div
            aria-label="Change Unit"
            class="MuiFormGroup-root MuiFormGroup-row"
            role="radiogroup"
          >
            <label
              class="MuiFormControlLabel-root"
            >
              <span
                aria-disabled="false"
                class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-5 MuiRadio-root MuiRadio-colorSecondary PrivateSwitchBase-checked-6 Mui-checked MuiIconButton-colorSecondary"
              >
                <span
                  class="MuiIconButton-label"
                >
                  <input
                    checked=""
                    class="PrivateSwitchBase-input-8"
                    name="Check unit"
                    type="radio"
                    value="Celcius"
                  />
                  <div
                    class="PrivateRadioButtonIcon-root-9 PrivateRadioButtonIcon-checked-11"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                      />
                    </svg>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root PrivateRadioButtonIcon-layer-10"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                      />
                    </svg>
                  </div>
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </span>
              <span
                class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
              >
                Celcius
              </span>
            </label>
            <label
              class="MuiFormControlLabel-root"
            >
              <span
                aria-disabled="false"
                class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-5 MuiRadio-root MuiRadio-colorSecondary MuiIconButton-colorSecondary"
              >
                <span
                  class="MuiIconButton-label"
                >
                  <input
                    class="PrivateSwitchBase-input-8"
                    name="Check unit"
                    type="radio"
                    value="Faranheit"
                  />
                  <div
                    class="PrivateRadioButtonIcon-root-9"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                      />
                    </svg>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root PrivateRadioButtonIcon-layer-10"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                      />
                    </svg>
                  </div>
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </span>
              <span
                class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
              >
                Faranheit
              </span>
            </label>
          </div>
        </fieldset>
      </div>
    </div>
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-grid-xs-12"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-align-items-xs-center MuiGrid-justify-xs-center MuiGrid-grid-xs-12 MuiGrid-grid-md-6"
      >
        <p
          class="sc-jSFjdj TTvdf"
        />
        <p>
          Thursday Jan 1
        </p>
        <div
          class="sc-iCoGMd hgXSwb"
        >
          <img
            height="75"
            src="http://localhost:3000/weatherIcon_5.svg"
            width="100"
          />
          <div
            class="sc-fujyAs dpeSXP"
          >
            <span>
               â„ƒ 
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-align-items-xs-center MuiGrid-justify-xs-center MuiGrid-grid-xs-12 MuiGrid-grid-md-6"
      >
        <div
          class="sc-gKAaRy IFDKQ"
        >
          <dl>
            <dt>
              Humidity
            </dt>
            <dd>
              <b />
            </dd>
            <br
              aria-hidden="true"
            />
            <dt>
              Visibility
            </dt>
            <dd>
              <b />
            </dd>
            <br
              aria-hidden="true"
            />
            <dt>
              Air Pressure
            </dt>
            <dd>
              <b>
                PA
              </b>
            </dd>
            <br
              aria-hidden="true"
            />
            <dt>
              Wind Speed
            </dt>
            <dd>
              <b>
                kmp
              </b>
            </dd>
          </dl>
        </div>
      </div>
    </div>
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-12"
      />
    </div>
  </div>
</div>
`;
